foundation-extensions/adobe-experience-platform-assurance/README.md

```diff
@@ -46,29 +46,56 @@ Follow these steps to add the install the extension in Experience Platform Launc
    ```
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS (AEP 3.x)" %}
 Add the library to your project via your [Cocoapods](https://cocoapods.org/pods/AEPAssurance) `Podfile`
 
 ```text
-pod 'ACPCore'
-pod 'AEPAssurance'
+pod 'AEPCore'
+pod 'AEPAssurance', '3'
```

Comment:
can we use the same notation ~> 3.0 ? this is also uniform with the Launch instructions